mode.c

```diff
@@ -224,6 +224,7 @@ void init_vm_mode(void) {
 	behavior_start(B_LEDS_PROX);
 	behavior_start(B_SOUND_BUTTON);
 	behavior_start(B_LEDS_MIC);
+	prox_disable_network();
```

Comment:
Your are right, I cannot reproduce it... A user tell me one time this problem and I was badly thinking I had the solution, but it is already correctly turn off in the exit mode...  And with the correct state on load this is now useless.